forem/forem
# Storybook: Add DocsPage and test out static content

## What type of PR is this? (check all applicable)


## Description

In this PR, I'm doing a few things:
- [x] Adding the DocsPage add-on
- [x] Updating the static pages (content), and making them into `mdx` pages

## Related Tickets & Documents
Closes https://github.com/forem/InternalProjectPlanning/issues/281

## QA Instructions, Screenshots, Recordings
To test Storybook locally on your machine, run these commands:

```
yarn
yarn storybook
```
Once you run `yarn storybook`, the localhost should open up in your default browser. For me, it was `localhost:6006`.

### Proposed changes
![Screen Shot 2020-11-16 at 1 18 40 PM](https://user-images.githubusercontent.com/816402/99310943-a3586880-2810-11eb-8851-9802b2b73bf5.png)

### Before
![Screen Shot 2020-11-16 at 1 19 02 PM](https://user-images.githubusercontent.com/816402/99310947-a5222c00-2810-11eb-9420-7562857fb875.png)


## Added tests?

- [x] No, and this is why: I'm not sure if we have tests for the updates that I'm suggesting.

## Added to documentation?
